test/browser/main.fmf

```diff
@@ -0,0 +1,13 @@
+summary:
+    Run browser integration tests on the host
+require:
+  - cockpit-podman
+  - cockpit-ws
+  - cockpit-system
+  - git
+  - libvirt-python3
```

Comment:
I'm also quite sure that this will need a few more dependencies, some qemu stuff. We can debug this while fixing packit tests.